plexus-compiler-its/pom.xml

```diff
@@ -3,9 +3,9 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.codehaus.plexus</groupId>
-    <artifactId>plexus-compiler</artifactId>
-    <version>2.8.9-SNAPSHOT</version>
+      <groupId>org.codehaus.plexus</groupId>
```

Comment:
nit why changing space?

Reply:
Sorry, I didn't intentionally change any of this and though that I had reverted all of the spurious editor "reformatting". I will attempt to remove it. I have whitespace diffs ignored by default in my IDE so didn't even notice.